### ISSUE TITLE
Add package-lock.json

### ISSUE THREAD

**Issue by calebeby** [cloudfour/stylelint-config-cloudfour#9]
It is currently missing `package-lock.json`

**Comment by spaceninja** [comment]
That's intentional — the recommendation is to not include package.lock for config packages:

[Lockfiles for apps, but not for packages](https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514)

[Stylelint debate about this](https://github.com/stylelint/stylelint/issues/3895)

**Comment by spaceninja** [comment]
Upon closer readings of those threads, I found some good counterpoints:

1) the biggest argument against package-lock was that it requires constantly updating dependencies. Which we get for free from Renovate.

2) [This blog post from the Yarn team about using lockfiles in libraries](https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/).

TL;DR: Listen to @calebeby ;)
